src/mo_frontend/parser.mly

```diff
@@ -674,6 +674,8 @@ exp_un(B) :
     { RetE(TupE([]) @? at $sloc) @? at $sloc }
   | RETURN e=exp(ob)
     { RetE(e) @? at $sloc }
+  | VAR COLON RETURN
```

Comment:
Is it possible to move this to `assertions.mly`, to avoid polluting the vanilla Motoko grammar?